core/src/api/api.rs

```diff
@@ -177,6 +179,7 @@ impl GraphAPI for GraphState {
 
 	/// Calculate the necessary page updates for all users graphs and return as a map of pages to
 	/// be updated and/or removed or added keys
+	#[log_result(Level::Error)]
```

Comment:
1. Only APIs that return a `Result` can be tagged
2. Missed `import_users_data` and `apply_actions` because they take a `&mut self` reference, which the macro implementation doesn't like. I'll add some manual logging to those.